ui/src/app/workflows/components/workflow-logs-viewer/workflow-logs-viewer.tsx

```diff
@@ -116,7 +186,20 @@ export const WorkflowLogsViewer = ({workflow, nodeId, initialPodName, container,
                 />{' '}
                 / <Autocomplete items={containers} value={selectedContainer} onSelect={setContainer} />
                 <span className='fa-pull-right'>
-                    <i className='fa fa-filter' /> <input type='search' defaultValue={logFilter} onChange={v => setLogFilter(v.target.value)} placeholder='Filter (regexp)...' />
+                    <div className='log-menu'>
+                        <i className='fa fa-filter' />{' '}
+                        <input type='search' defaultValue={logFilter} onChange={v => setLogFilter(v.target.value)} placeholder='Filter (regexp)...' />
+                        <i className='fa fa-globe' />{' '}
+                        <Autocomplete
+                            items={filteredTimezones}
+                            value={uiTimezone}
+                            onChange={v => setUITimezone(v.target.value)}
+                            onSelect={tz => {
+                                setUITimezone(tz);
+                                setTimezone(tz);
+                            }}
```

Comment:
If I'm understanding correctly, it looks like setUITimezone() would be called automatically when timezone changes (lines 86-88), so maybe setUITimezone() wouldn't need to be explicitly called on line 198?

Reply:
Ah good pickup, this isn't needed anymore. I forgot to update this line after introducing the useEffect.